yopeso/yopeso-coronavirushackaton
# 1082799 create user account

## Ready State
**Ready**

## Summary
- update migration and seeder
- added new user registration form
- added authorization on controller level

## Ticket
[1082799](https://portal.yopeso.com/task/1082799) 

## Testing
- login using `global@gep.yopeso.com` and access `/create-user`, you should be able to create a new user and assign account type and country as well.
- create a normal user account and try to access `/create-user`, you should be redirect to the listing page
- make sure to run migration and seeder first
